services/ingestion/etl/agri/csv/test/unit/lib/transform.spec.js

```diff
@@ -2,10 +2,14 @@
  * @jest-environment node
  */
 
+import moment from 'moment-timezone';
+
 import mapper from '../../../src/lib/transform';
 import testRecord from '../../stubs/record.json';
 
-describe(`DG BUDG CSV transformer`, () => {
+describe(`DG AGRI CSV transformer`, () => {
+  moment.tz.setDefault('CET');
```

Comment:
What benefit does `moment.tz.setDefault` bring?

Reply:
It's preventing drone CI and other systems which are not in our TZ to use our TZ. Inspiration came from [this article](https://medium.com/front-end-hacking/jest-snapshot-testing-with-dates-and-times-f3badb8f1d87).

Reply:
Thanks! Makes sense. Just out of curiosity, did we have any issue with dates so far? Or is `moment-timezone` needed because we now use `moment`?